src/pyskindose/rdsr_parser.py

```diff
@@ -31,10 +30,10 @@ def rdsr_parser(data_raw: pydicom.FileDataset) -> pd.DataFrame:
             # Declare temporary dictionary
             data_parsed_dict = dict()
 
-            # Save manufacturer model name
-            data_parsed_dict["model"] = data_raw.ManufacturerModelName\
-                .replace(" ", "").replace("-", "").replace("(", "")\
-                .replace(")", "").replace(".", "")
+            # Save manufacturer, and manufacturer model name
+            data_parsed_dict["Manufacturer"] = data_raw.Manufacturer
+            data_parsed_dict["ManufacturerModelName"] = \
```

Comment:
Hard coded strings. Extract to constants?

Reply:
done